From 16dbe5168224b9f8a2b193f014f3dbf7e9604516 Mon Sep 17 00:00:00 2001
From: Daniel Murnane <dmurnane@eitccorp.com>
Date: Fri, 1 May 2020 08:19:28 -0400
Subject: [PATCH] Notify systemd on sentinel startup

---
 src/server.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/src/server.c b/src/server.c
index 659604ef363..7ab7c39fd0a 100644
--- a/src/server.c
+++ b/src/server.c
@@ -5111,6 +5111,10 @@ int main(int argc, char **argv) {
     } else {
         InitServerLast();
         sentinelIsRunning();
+        if (server.supervised_mode == SUPERVISED_SYSTEMD) {
+            redisCommunicateSystemd("STATUS=Ready to accept connections\n");
+            redisCommunicateSystemd("READY=1\n");
+        }
     }
 
     /* Warning the user about suspicious maxmemory setting. */
